pipeline-model-definition/src/test/resources/postStage/unsuccessful.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
```

Comment:
Same thing as previously. Since this is a new file, the copyright date should probably be 2018.